common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -369,6 +372,42 @@ open class MusicService : MediaBrowserServiceCompat() {
                 }
             }
         }
+
+        override fun onPlayerError(error: ExoPlaybackException) {
+            when (error.type) {
+                // If the data from MediaSource object could not be loaded the Exoplayer raises
+                // a type_source error.
+                // An error message is printed to UI via Toast message to inform the user.
+                ExoPlaybackException.TYPE_SOURCE -> {
+                    Toast.makeText(
```

Comment:
We're only displaying a `Toast` for the `TYPE_SOURCE` error. Is it worth moving the `Toast` outside this block so that it's displayed for all types of error?

Reply:
well, I can write a generic text message for those errors but in my opinion, the "_error_media_not_found_" message is not too specific and doesn't fit well the other situation. Please tell me what you think about it.

Reply:
Sorry, I didn't explain that well. I mean: 

- Inside the `when` block build a specific message for each type of error
- Show the Toast with that specific message outside the `when` block

That way the user will be informed if there's _any_ type of error, not just `TYPE_SOURCE`

Reply:
Forgive my laziness :) I don't think it is necessary to display every type of error reported to the end-user. I have created a generic one for most of them but I left TYPE_SOURCE unchanged. Do you agree?